src/parser/estree_translator.ml

```diff
@@ -966,6 +967,7 @@ end with type t = Impl.t) = struct
       | Literal.Boolean b -> bool b
       | Literal.Null -> null
       | Literal.Number f -> number f
+      | Literal.BigInt _ -> null
```

Comment:
It's my impression that this pattern match will never be hit. So it might be a little safer to instead write something like

  | Literal.BigInt _ -> failwith "We should not create Literal nodes for bigints"